packages/cover-traffic-daemon/tsconfig.json

```diff
@@ -2,9 +2,10 @@
   "extends": "../../tsconfig.json",
   "compilerOptions": {
     "outDir": "lib",
-    "rootDir": "src"
+    "rootDir": "src",
+    "resolveJsonModule": true
```

Comment:
There was discussion and @robertkiel advised against it as I recall

Reply:
Any particular reason why?

Reply:
https://github.com/hoprnet/hoprnet/pull/2935#issuecomment-972846451

Reply:
Solved in f7f68af875d39012c5f3cf59c39e858f8c608dda